docs/examples/openapi/customize_operation_class.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass, field
+from typing import Dict, List, Optional
+
+from litestar import Litestar, MediaType, Request, post
+from litestar.exceptions import HTTPException
+from litestar.openapi.spec import OpenAPIMediaType, OpenAPIType, Operation, RequestBody, Schema
+from litestar.status_codes import HTTP_400_BAD_REQUEST
+
+
+@dataclass
+class CustomOperation(Operation):
+    """Custom Operation class which includes a non-standard field which is part of an OpenAPI extension."""
+
+    x_code_samples: Optional[List[Dict[str, str]]] = field(default=None, metadata={"alias": "x-codeSamples"})
+
+    def __post_init__(self) -> None:
+        self.tags = ["ok"]
+        self.description = "Requires OK, Returns OK"
+        self.request_body = RequestBody(
+            content={
+                "text": OpenAPIMediaType(
+                    schema=Schema(
+                        title="Body",
+                        type=OpenAPIType.STRING,
+                        example="OK",
+                    )
+                ),
+            },
+            description="OK is the only accepted value",
+        )
+        self.x_codeSamples = [
```

Comment:
I know it has been merged, but I'd like to make a note that this should be 
`self.x_code_samples`

Reply:
That's true. Fix PR welcome